test/e2e/system_reset_test.go

```diff
@@ -97,4 +98,33 @@ var _ = Describe("podman system reset", func() {
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToStringArray()).To(BeEmpty())
 	})
+
+	It("podman system reset, check podman.service", func() {
+		SkipIfRemote("system reset not supported on podman --remote")
+		SkipIfNotRootless("systemctl only works here with --user")
+		SkipIfInContainer("systemd does not run in the containerized tests")
+
+		sys, err := exec.LookPath("systemctl")
+		Expect(err).Should(BeNil())
```

Comment:
The test should not hard error for this. What if someone wants to run it on a non systemd environment?

Reply:
I think we decided that this use case is just to restart the service on machines that have systemd. If we want to use dbus api I think that involves another discussion. Usually with system reset we operate under the logic of "you should know what you are doing when you are using this" @rhatdan @vrothberg WDYT?

Reply:
sure but why do you error out if systemctl is not found? if there is no systemd you should just skip the test

Reply:
that sgtm, I will do that